website/docs/r/windows_virtual_machine_scale_set.html.markdown

```diff
@@ -264,6 +266,10 @@ A `data_disk` block supports the following:
 
 ~> **NOTE:** Disk Encryption Sets are in Public Preview in a limited set of regions
 
+* `disk_iops_read_write` - (Optional) Specifies the Read-Write IOPS for this Data Disk. Only settable for UltraSSD disks.
```

Comment:
Would this make more sense as
```suggestion
* `ultra_ssd_disk_iops_read_write` - (Optional) Specifies the Read-Write IOPS for this Data Disk. Only settable for UltraSSD disks.
```

to make it clear it only applies to ultra ssds?

Reply:
Well, I would love to do this change, but these two attributes already exist in the `azurerm_managed_disk` resource, would this cause any incosistency?

Reply:
We should leave these consistent with elsewhere in the provider, even though they only have meaning in one circumstance.

Reply:
Should eventually change both to `ultra_ssd` then - i'll just add a comment